tools/aqrdev

```diff
@@ -620,11 +630,16 @@ def _get_deployment_status(name: str) -> Dict[str, List[Tuple[str, str]]]:
 
     path: Path = _get_deployment_path(name)
     if not path.exists():
-        raise DeploymentPathNotFoundError()
+        raise DeploymentPathNotFoundError(
+            "deployment not found",
+            rc=errno.ENOENT
+        )
 
     vagrantfile = path.joinpath("Vagrantfile")
     if not vagrantfile:
-        raise DeploymentNotFinishedError()
+        raise DeploymentNotFinishedError(
+            "deployment not finished", rc=errno.ENOENT
```

Comment:
I would almost prefer to reverse these args (e.g. errno appear first), but we'd need to extend the Error class todo that.

wdyt if I were to rework this via a PR #235 ?

Reply:
> wdyt if I were to rework this via a PR #235 ?

sounds good to me!